### PR TITLE
report table tweaks CVR-687

### PR DESCRIPTION
**Pull request by hobbitronics**
image

fixed Total and removed rate
image

[CVR-687](https://itse.youtrack.cloud/issue/CVR-687/Report-Per-transaction-rates-and-renewals)

- moved rate to table
- don't show duplicate status in table

### CODE REVIEW

File: rollup.config.js
Comment (forevermatt) [inline rollup.config.js:1]: These look like the same changes that were in another PR. Do you want to remove them from this PR to avoid merge conflicts?

**Review by forevermatt — APPROVED** [review]
Approved, since the comments I added are simple things to address (or not, as you see fit). Thanks! 👍

File: src/pages/policies/[policyId]/report.svelte [hunk 1]
```diff
@@ -161,10 +159,16 @@ th {
...
+            {#if entry.status_before && entry.status_before != entry.status_after}
+              <td>{entry.status_before}</td>
+              <td>-></td>
+              <td>{entry.status_after}</td>
+            {:else}
+              <td colspan="3" class="text-align-center">{entry.status_after}</td>
+            {/if}
             <td>{entry.type}</td>
+            <!-- Todo get rate from entry when available -->
+            <td>{reportData.premium_rate * 100}%</td>
```
Comment (briskt) [inline src/pages/policies/[policyId]/report.svelte:171]: Probably better to remove this until it can be added correctly.

**Comment by merchako** [comment]
why remove the rate?

**Comment by briskt** [comment]
> why remove the rate? Is this a hot fix?

At this point, I'm not sure we'll be able to do it. See [CVR-692](https://itse.youtrack.cloud/issue/CVR-692) for the explanation.

**Pull request merged**